charts/vsphere-cpi/v1.0.0/values.yaml

```diff
@@ -1,11 +1,20 @@
 vCenter:
   host: ""
-  insecureflag: "1"
-  port: "443"
+  port: 443
+  insecureFlag: true
```

Comment:
Is there an impact from changing `1` to `true`?

Reply:
Good catch. I think so cause the default said "1" ...

Reply:
https://vmware.github.io/vsphere-storage-for-kubernetes/documentation/existing.html#template-config-file yes. We should change back to 1 or use helm template functions to translate true -> 1 and false -> 0

Reply:
I don't think so, QA tested CSI with true instead of and their doc actually says <true or false>

Reply:
I assume this was from the copy/paste from CSI, but we should use "1" if that's what it used to be.

Reply:
ok, reverting